From 859c3fd09c1b8c30998b03da9002e7f00b6670cc Mon Sep 17 00:00:00 2001
From: Aleksei Bavshin <alebastr89@gmail.com>
Date: Wed, 22 Jan 2025 00:50:53 -0800
Subject: [PATCH imv] Link to the common ICU library

Both the utext and the ubrk APIs are parts of the common library, so
icu-uc should be the right dependency to use.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index ea3a653..9fa1b15 100644
--- a/meson.build
+++ b/meson.build
@@ -42,7 +42,7 @@ endif
 
 _unicode = get_option('unicode')
 if _unicode == 'icu'
-  unicode_lib = dependency('icu-io')
+  unicode_lib = dependency('icu-uc')
   add_project_arguments('-DIMV_USE_ICU', language: 'c')
 elif _unicode == 'grapheme'
   unicode_lib = cc.find_library('grapheme')
-- 
2.48.1

